src/apiServices/AuthService.ts

```diff
@@ -36,6 +36,7 @@ export class AuthService {
             return tokens
         } catch (err) {
             errorHandler(err)
+            throw err
```

Comment:
this is just a workaround to avoid the `function return type can not be undefined` message. Basically the  second throw never gets reached (since `errorHandler` already throws) but TS is not smart enough to know that `errorHandler` always throws an error if called

Reply:
Ah, in that case, lets be a bit more consistent about whether we add `| void` to the type or add the `throw err`. I see different things used in different places.